src/components/club/ClubHeading.tsx

```diff
@@ -20,16 +20,27 @@ export default function ClubHeading({ info }: ClubHeadingProps) {
     location,
     regularMeeting,
     profileImage,
-    startRecruitPeriod,
-    endRecruitPeriod,
-    formUrl,
+    startDate,
+    endDate,
+    formId,
   } = info;
 
   const imageSrc = profileImage?.originUrl ? profileImage?.originUrl : Admin;
   const { data } = useAllClubs();
   const isRecruit =
     data?.data.find((club) => club.name === name)?.recruitStatus ===
-      '모집 중' && formUrl;
+      '모집 중' && formId;
+
+  const router = useRouter();
+
+  interface MoveToApply {
+    (id: number): void;
+  }
+
+  const moveToApply: MoveToApply = (id) => {
+    router.push(`/apply/${id}`);
+  };
```

Comment:
수정요구사항은 아니지만 함수 시그니처를 별도로 관리하셨는데, 기존 코드에서는 매개변수에서 타입을 호출하는 방식을 일관되게 사용하고 있어용. 참고차원에서 공유드립니당
```suggestion
const moveToApply = (id: number) => {
    router.push(`/apply/${id}`);
  };
```